src/Common/tests/System/Net/Http/Http2LoopbackServer.cs

```diff
@@ -611,11 +614,11 @@ public override async Task<HttpRequestData> HandleRequestAsync(HttpStatusCode st
 
             if (content == null)
             {
-                await SendResponseHeadersAsync(streamId, true, statusCode, headers);
+                await SendResponseHeadersAsync(streamId, true, statusCode, false, headers);
             }
             else
             {
-                await SendResponseHeadersAsync(streamId, false, statusCode, headers);
+                await SendResponseHeadersAsync(streamId, false, statusCode, false, headers);
```

Comment:
Nit: for both the existing and new Boolean args, can you please use named arguments?  Otherwise, it's hard to know what they mean.